packages/shared/ReactTypes.js

```diff
@@ -192,6 +192,13 @@ export type ReactCallSite = [
 
 export type ReactStackTrace = Array<ReactCallSite>;
 
+export type ReactFunctionLocation = [
+  string, // function name
+  string, // file name TODO: model nested eval locations as nested arrays
+  number, // enclosing line number
+  number, // enclosing column number
```

Comment:
Like it's not the enclosing of the function location but more like what enclosing means in the context of a callsite. I.e. function location.